pkg/apiserver/middlewares/v1/api_key.go

```diff
@@ -89,6 +91,22 @@ func (a *APIKey) MiddlewareFunc() gin.HandlerFunc {
 			}
 		}
 
+		useragent := strings.Split(c.Request.UserAgent(), "/")
+
+		if len(useragent) != 2 {
+			log.Errorf("bad user agent from ('%s') : %s", useragent, c.ClientIP())
+			c.JSON(http.StatusForbidden, gin.H{"message": "bad user agent"})
```

Comment:
maybe it's a bit aggressive to return a 403 if we don't like the user-agent format ? can't we just silently skip it ?